mmedit/models/editors/disco_diffusion/clip_ext.py

```diff
@@ -0,0 +1,72 @@
+    | ViTL14                        | clip      | name='ViT-L/14',                  jit=False                  | # noqa
+    | ViTL14_336px                  | clip      | name='ViT-L/14@336px',            jit=False                  | # noqa
+    | RN50                          | clip      | name='RN50',                      jit=False                  | # noqa
+    | RN50x4                        | clip      | name='RN50x4',                    jit=False                  | # noqa
+    | RN50x16                       | clip      | name='RN50x16',                   jit=False                  | # noqa
+    | RN50x64                       | clip      | name='RN50x64',                   jit=False                  | # noqa
+    | RN101                         | clip      | name='RN101',                     jit=False                  | # noqa
+    | ViTB32_laion2b_e16            | open_clip | name='ViT-B-32',                  pretrained='laion2b_e16'   | # noqa
+    | ViTB32_laion400m_e31          | open_clip | model_name='ViT-B-32',            pretrained='laion400m_e31' | # noqa
+    | ViTB32_laion400m_32           | open_clip | model_name='ViT-B-32',            pretrained='laion400m_e32' | # noqa
+    | ViTB32quickgelu_laion400m_e31 | open_clip | model_name='ViT-B-32-quickgelu',  pretrained='laion400m_e31' | # noqa
+    | ViTB32quickgelu_laion400m_e32 | open_clip | model_name='ViT-B-32-quickgelu',  pretrained='laion400m_e32' | # noqa
+    | ViTB16_laion400m_e31          | open_clip | model_name='ViT-B-16',            pretrained='laion400m_e31' | # noqa
+    | ViTB16_laion400m_e32          | open_clip | model_name='ViT-B-16',            pretrained='laion400m_e32' | # noqa
+    | RN50_yffcc15m                 | open_clip | model_name='RN50',                pretrained='yfcc15m'       | # noqa
+    | RN50_cc12m                    | open_clip | model_name='RN50',                pretrained='cc12m'         | # noqa
+    | RN50_quickgelu_yfcc15m        | open_clip | model_name='RN50-quickgelu',      pretrained='yfcc15m'       | # noqa
+    | RN50_quickgelu_cc12m          | open_clip | model_name='RN50-quickgelu',      pretrained='cc12m'         | # noqa
+    | RN101_yfcc15m                 | open_clip | model_name='RN101',               pretrained='yfcc15m'       | # noqa
+    | RN101_quickgelu_yfcc15m       | open_clip | model_name='RN101-quickgelu',     pretrained='yfcc15m'       | # noqa
+
+    An example of a ``clip_modes_cfg`` is as follows:
+    .. code-block:: python
+
+        clip_models_cfg = [
+            dict(type='ClipWrapper', clip_type='clip', name='ViT-B/32', jit=False),
+            dict(type='ClipWrapper', clip_type='clip', name='ViT-B/16', jit=False),
+            dict(type='ClipWrapper', clip_type='clip', name='RN50', jit=False)
+        ]
+
+    Args:
+        clip_type (List[Dict]): The original source of the clip model. Whether be
+            ``clip`` or ``open_clip``.
+    """
+
+    def __init__(self, clip_type, *args, **kwargs):
```

Comment:
Can we use `setting` to init clip model directly? e.g.`dict(type='ClipWrapper', setting='RN50')`

Reply:
Do you mean we hardcode certain clip settings? I need to check whether `name` or `model_name` is the only key decide the clip models.

Reply:
Maybe we can keep unchanged and see it later